python-packages/order_utils/README.md

```diff
@@ -0,0 +1,47 @@
+API documentation hosted at [https://0x.readthedocs.io/projects/order-utils/en/latest/](https://0x.readthedocs.io/projects/order-utils/en/latest/)
+
+## Development
+
+### Edit/test cycle
+
+`pip install -e .[dev]`
+
+`./setup.py build`
+
+`./setup.py build_sphinx`
+
+`./setup.py test`
+
+`./setup.py lint`
+
+`./setup.py clean --all`
+
+See `./setup.py --help-commands` for more info.
+
+### Publishing
+
+`./setup.py sdist bdist_wheel` will build the distribution into the `dist` folder. Once you're happy with it, you can use `./setup.py` to upload it to the package index, as described below.
+
+If you're just experimenting, and want to try out a publish without committing to it, consider using a "staging package;" that is, rename the package, in `setup(name=...)` in `setup.py`, to something like "0x-order-utils-staging". The reason is that PyPI will never let you re-publish to an already-used file name (which includes the package name and version), even if you delete that published version from the repo. With an alternate package name, you can increase the version number (required for a successful upload) for every little experiment, without "stealing" version numbers from the main package name.
```

Comment:
Nit: Intentional `staging package;"`?

Reply:
Hm... Since we don't expect end-developers to publish this library, let's move all sections here related to publishing to a Quip doc in the `documentation` folder about how publish the Python package.

Reply:
Yes that was intentional. Looks weird now that you mention it, but it's grammatically correct 🙂.